third-party/java/aosp/src/com/android/manifmerger/MergingReport.java

```diff
@@ -322,6 +322,8 @@ Builder addMessage(@NonNull SourceFilePosition sourceFilePosition,
                 case WARNING:
                     mHasWarnings = true;
                     break;
+		default:
+		    throw new AssertionError();
```

Comment:
I think most of these are deliberate, and should be `default: // fall out` instead of `default: throw new AssertionError();`.